scripts/test.ts

```diff
@@ -0,0 +1,40 @@
+import { setConfigMatcher } from "./models/setConfig.ts";
```

Comment:
Delete this file, we don't need to copy this across. I need to find the original and delete. I did it for testing the shapes of the configs

Reply:
Removed